src/Components/ShoppingCart.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+class ShoppingCart extends React.Component {
+  render() {
+    return (
+      <p data-testId="shopping-cart-empty-message">Seu carrinho está vazio</p>
+    );
+  }
+}
+
+export default ShoppingCart;
```

Comment:
O que acha de adicionar uma linha aqui? 
```suggestion
export default ShoppingCart;

```